util/options.hh

```diff
@@ -478,6 +481,9 @@ struct font_options_t : option_group_t
   hb_variation_t *variations;
   unsigned int num_variations;
   int default_font_size;
+  int x_ppem;
+  int y_ppem;
+  int ptem;
```

Comment:
The API takes ptem as float. Please make float and update argparsing to accept float.